bundles/org.openhab.binding.astro/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If there isn't any strong reason, I would prefer to keep the add-ons in separate folders by type (binding/transform/etc...) and not have them all as a flat list under `bundles`.

Reply:
What's different from core here?

Reply:
The new location is `bundles/org.openhab.binding.astro`, not `bundles/binding/org.openhab.binding.astro`.

Reply:
Ah, sorry, you meant the difference to openhab-core?
Well, the difference is that we have a HUGE list of bundles here and that we have well defined types that can be cleanly separated. This is not really the case in openhab-core.

Reply:
IIRC we had a discussion on the core migration and we agreed to use a flat list there.
https://github.com/openhab/openhab-core/pull/467#issuecomment-455995009

Now, what's the argument to do it different here?

Reply:
The question is rather, what is the strong reason to change the current structure of this repo?

Reply:
There is nothing "strong".

We commented that four days ago:
https://github.com/openhab/openhab-core/pull/589#issuecomment-467426669

I invested my saturday to implemented what we talked about, just because I would like to move things forward.
Now after the work has done, you suggest to change it to another layout.

I am not strong about the layout, and I am happy to accept PRs that change the layout on my branch.

Reply:
In openHAB Core all integration tests are part of the `itests` directory.
Every integration we use contains some configuration that is equal to all the other ones.
I managed it put all that stuff in `itests/itests-common.bndrun`.
To speed up things the indices for the most bundles are not generated for every integration tests again and again on build time but we reference to a global one.
Please have a look at that file.
To reference that locations I used relative URLs and the URLs needs to be realtive to the integration test that is executed. So, the one that includes the common itests file.
If the integration tests are in separate sub-directories that depth is not always the same, that is not possible anymore.
We need to drop the itests-common and put that stuff in every itests bndrun.
If we would like to change some global stuff in future (e.g. the standalone repos, the used OSGi framework, ...), we cannot change that single file anymore but we have to change all the integration tests' bndrun files.

In the migration phase we need to use two reactors, one for bnd and one for tycho.
If we keep the bundles (not itests) at the old place, it will be very tricky to get this running) as the parents references at least one reactor. So we need also for every parent between the bundle and the main reactor POM a tycho and a bnd version.
We could keep the directory layout for the addons -- just in another base directory (e.g. `bundles` instead of `addons`), but then it does not really fit to use a flat layout for the integration tests in `itests` and a hierachical in `bundles`.
IMHO the flat layout is very simple. But the addon in `bundles` and its integration test in `itests`.

Suggestions?
Are the choosen GA okay?

Reply:
> We commented that four days ago:
> openhab/openhab-core#589 (comment)
>
> I invested my saturday to implemented what we talked about, just because I would like to move things forward.
> Now after the work has done, you suggest to change it to another layout.

Sorry, I didn't follow the archetype discussion, thus I wasn't aware of that discussion. As you had agreed with @wborn and @davidgraeff already on the flat structure, this is all fine - I did't want to interfere here, sorry!